src/index.html

```diff
@@ -10,8 +10,8 @@
 </head>
 <body class='sans-serif'>
   <header class='pv3 ph2 ph3-l bg-navy cf'>
-    <a href="https://multiformats.io/" title='ipfs.io'>
-      <img src="https://ipfs.io/images/ipfs-logo.svg" class='v-mid' style='height:50px' />
+    <a href="https://ipfs.io/ipns/multiformats.io/" title='ipfs.io'>
```

Comment:
Hm, is there a specific reason for linking to not the canonical version? IPNS tends to be a bit slower than just using DNS + IPFS hashes

Reply:
I did not know that... I'll change them back.